src/bhavana_coding_exam_1/AgeDifference.java

```diff
@@ -0,0 +1,30 @@
+package bhavana_coding_exam_1;
+/*Program 2 : 
+WAP to provide the difference between a Younger and an elder member of the family. 
+Age is provided in the form of an array.
+
+Sample Input: 7 12 56 62 25
+
+Sample Output: 55 */
+
+
+
+public class AgeDifference {
+
+	int[] age= {7,12,56,62,25};
+	int difference;
+	int outputIndex=0;
+	
+	void differenceOfAge() {
+		for(int index=0;index<age.length;index++) {
+			difference=age[3]-age[0];
```

Comment:
what if number got change in array, this will not work @BhavanaShinde 
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files